website/client-old/js/controllers/hallCtrl.js

```diff
@@ -1,16 +1,56 @@
 "use strict";
 
-habitrpg.controller("HallHeroesCtrl", ['$scope', '$rootScope', 'User', 'Notification', 'ApiUrl', 'Hall',
-  function($scope, $rootScope, User, Notification, ApiUrl, Hall) {
+habitrpg.controller("HallHeroesCtrl", ['$scope', '$rootScope', 'User', 'Notification', 'ApiUrl', 'Hall', 'Content',
+  function($scope, $rootScope, User, Notification, ApiUrl, Hall, Content) {
     $scope.hero = undefined;
     $scope.currentHeroIndex = undefined;
     $scope.heroes = [];
+    $scope.allItemPaths = getAllItemPaths();
 
     Hall.getHeroes()
       .then(function (response) {
         $scope.heroes = response.data.data;
       });
 
+    function _getFormattedItemReference (pathPrefix, keys, values) {
+      var finishedString = '\n'.concat('path: ', pathPrefix, ', ', 'value: {', values, '}\n');
+
+      _.each(keys, (key) => {
+        finishedString = finishedString.concat('\t', pathPrefix, '.', key, '\n');
+      });
+      return finishedString;
+    }
+
+    function getAllItemPaths () {
+      var content = Content;
```

Comment:
What's the reason for assigning this variable? Can't you just use `Content` everywhere?